### ISSUE TITLE
Newer firmware required for Shelly i3 but no newer firmware available

### ISSUE THREAD

**Issue by Borrel88** [bieniu/ha-shellies-discovery#152]
When looking into the home asssitant logs, I get the following erro:

```
2021-09-29 12:41:01 ERROR (SyncWorker_3) [homeassistant.components.python_script.shellies_discovery.py] Error executing script: Firmware dated 20210720 is required, please update your device shellyix3-68C63AF9EF1E
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/python_script/__init__.py", line 222, in execute
    exec(compiled.code, restricted_globals)
  File "shellies_discovery.py", line 571, in <module>
ValueError: Firmware dated 20210720 is required, please update your device shellyix3-68C63AF9EF1E
```

So I decided to update my Shelly i3 Device.
But, I already got the newest version installed, which is: *20210429-102402/v1.10.4-g3f94cd7*

Is this a bug on the script's side, or should there be a newer firmware available which I can't install?

**Comment by cliffkujala** [comment]
I just installed my first i3, updated the firmware to latest available, then tried to add to home assistant via a working ha-shellies-discovery setup and I receive the same error in my HA core log.  

My Shelly i3 says latest available firmware is 20200421-183805/master@d834f204 

Maybe there is just a problem with i3's pulling down the correct updated version of firmware?

**Comment by Borrel88** [comment]
I created a ticket on the Shelly website to confirm if there should be a newer firmware version for the i3 devices.
I will let you know if I got a response

**Comment by cliffkujala** [comment]
I think this is a known issue with the i3 if they shipped with 20200421-183805/master@d834f204 firmware.  I did a manual OTA update to a later version and then I was fine.

**Comment by Borrel88** [comment]
Just received a reply from shelly on my ticket.
They explained to me the way to update manually to a newer version, of course they are not saying there is a problem with the i3 devices.

But everything is working fine now, closing this because it is working
